keps/prod-readiness/sig-node/3619.yaml

```diff
@@ -4,3 +4,5 @@
 kep-number: 3619
 alpha:
   approver: "@johnbelamaric"
+beta:
```

Comment:
no needed for this PR, now?

Reply:
@johnbelamaric Sorry for taking the time to do this PRR. As we postpone its beta promotion to v1.33 or later(depending on containerd v2 situation). So, I think you don't need to review this KEP for now (at least next until v1.33 release cycle.